src/App.js

```diff
@@ -8,12 +8,25 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies,
+      movies: [],
       onMainPage: true,
       singleMovie: {},
+      error: null
     }
   }
 
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+    .then(data => {
+      this.setState({ movies: data.movies })
+    })
+    .catch(err => {
+      console.log(err)
+      this.setState({ error: err })
```

Comment:
I was just thinking that maybe if the server is down we render a different message that says "Try back later?" That way they don't keep refreshing if it's a server error? Let's brainstorm on how we want to handle this.